R/install_perl_on_mac.R

```diff
@@ -0,0 +1,18 @@
+#' Set Perl up on macOS
```

Comment:
This can be used on any non-Windows, unix-derived system that we support (mac, linux, BSD variants etc).

Reply:
Just dated this one!